src/Redirects.php

```diff
@@ -86,6 +86,18 @@ public function seeTemporaryRedirectBetween($oldUrl, $newUrl)
         $this->seeRedirectBetween($oldUrl, $newUrl, 307);
     }
 
+	/**
+	 * Check that the specified HTTP Status is returned with the correct Location URL.
+	 *
+	 * @param string $oldUrl     Relative or absolute URL that should be redirected.
+	 * @param string $newUrl     Relative or absolute URL of redirect destination.
+	 * @param int    $statusCode HTTP Status code to check against.
+	 */
+	public function seeOtherRedirectBetween($oldUrl, $newUrl, $statusCode)
```

Comment:
When I suggested `seeOtherRedirectBetween()`, it was because the the 303 HTTP Status Code is called _Other Redirect_.

As such, line 98 would be hard-coded to 303, like the others are 307 and 301.

As it stands, this method is just a public wrapper to the private `seeRedirectBetween()`, so the simpler solution would be to change the access of `seeRedirectBetween()` so it's usable outside of the class.

If this happens, then the documentation should also be updated as well, including noting that the existing `see*RedirectBetween()` methods would now just be convenience methods.

Reply:
Ah, my mistake. I think your proposed solution makes the most sense so I will go ahead with making `seeRedirectBetween()` public and updating the documentation accordingly.